src/ansys/workbench/core/workbench_client.py

```diff
@@ -381,26 +381,43 @@ def download_file(self, file_name, show_progress=True, target_dir=None):
             pbar.close()
         return file_name
 
-    def download_project_archive(self, archive_name, show_progress=True):
+    def download_project_archive(
+        self, archive_name, include_solution_result_files=True, show_progress=True
+    ):
         """Create and download the project archive.
 
         Parameters
         ----------
         archive_name : str
             Name of the project archive to use, without the file extension.
+        include_solution_result_files : bool, default: True
+            Whether to include solution/result files in the archive.
         show_progress : bool, default: True
             Whether to show a progress bar during the download.
         """
         if not re.match(r"^\w+$", archive_name):
             logging.error("archive name should contain only alphanumeric characters")
             return
         script = f"""import os
+import json
+successful = False
 wd = GetServerWorkingDirectory()
 if os.path.basename(GetProjectFile()).StartsWith("wbnew."):
     Save(FilePath=os.path.join(wd, "{archive_name}.wbpj"), Overwrite=True)
-Archive(FilePath=os.path.join(wd, "{archive_name}.wbpz"))
+Archive(FilePath=os.path.join(wd, "{archive_name}.wbpz"),
+    IncludeSkippedFiles={include_solution_result_files})
+successful = True
+wb_script_result =json.dumps(successful)
 """
-        self.run_script_string(script)
+        archive_created = self.run_script_string(script)
+        if not archive_created:
+            logging.error(
+                (
+                    "failed to create the project archive. "
+                    "make sure that the solver PyAnsys sessions are closed."
```

Comment:
```suggestion
                    "Failed to create the project archive. "
                    "Make sure that the solver PyAnsys sessions are closed."
```
You sentence construction--which means start with an initial cap. Otherwise, it is fine.